consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/api/CMCApi.java

```diff
@@ -64,4 +66,12 @@ WarrantResponse warrantRequest(@RequestHeader(AUTHORIZATION) String authorizatio
                                    @RequestHeader(SDT_REQUEST_ID) String sdtRequestId,
                                    @RequestBody WarrantRequest warrantRequest);
 
+    @PostMapping("/requestWarrant")
```

Comment:
Should the path be "/requestJudgmentAndWarrant"?

Reply:
changed